contracts/mocks/ERC1155PausableMock.sol

```diff
@@ -16,14 +16,13 @@ contract ERC1155PausableMock is ERC1155Mock, ERC1155Pausable {
         _unpause();
     }
 
-    function _beforeTokenTransfer(
-        address operator,
+    function _update(
         address from,
         address to,
         uint256[] memory ids,
         uint256[] memory amounts,
         bytes memory data
-    ) internal override(ERC1155, ERC1155Pausable) {
-        super._beforeTokenTransfer(operator, from, to, ids, amounts, data);
+    ) internal virtual override(ERC1155, ERC1155Pausable) {
```

Comment:
mocks function don't need to be virtual, do they? we don't plan on overriding them

```suggestion
    ) internal override(ERC1155, ERC1155Pausable) {
```